lib/sync/util.js

```diff
@@ -7,7 +7,7 @@ var _ = require('underscore');
 var SYNC_LOGGER = 'SYNC';
 var loggers = {};
 
-var generateHash = function (plainText) {
+function generateHash(plainText) {
```

Comment:
@grdryn I don't think we have such a thing. But personally I do use both, and I am not sure if we need to specify which one to use. But instead, we should set a rule that always declare the function BEFORE invoking it. This will avoid the "hoist" problem, and makes it easier to understand as well.